region/crateria/central/Climb.json

```diff
@@ -2130,21 +2130,44 @@
       "name": "Space Pirate Farm",
       "requires": [
         "h_ZebesIsAwake",
-        {"resetRoom": {
-          "nodes": [1, 5]
-        }},
-        {"refill": ["Energy", "Missile"]},
         {"or": [
           "h_ClimbWithoutLava",
           "h_lavaProof",
           {"obstaclesNotCleared": ["B"]}
+        ]},
+        {"or": [
+          {"and": [
+            {"resetRoom": {"nodes": [1]}},
+            {"cycleFrames": 2350}
+          ]},
+          {"and": [
+            {"resetRoom": {"nodes": [2]}},
+            {"or": [
+              "h_canUseMorphBombs",
+              "ScrewAttack"
+            ]},
+            {"cycleFrames": 2270}
+          ]},
+          {"and": [
+            {"resetRoom": {"nodes": [3]}},
+            "h_canUseMorphBombs",
+            {"cycleFrames": 2695}
+          ]},
+          {"and": [
+            {"resetRoom": {"nodes": [4]}},
+            "h_canUseMorphBombs",
+            {"cycleFrames": 2480}
+          ]},
+          {"and": [
+            {"resetRoom": {"nodes": [5]}},
+            {"cycleFrames": 1905}
```

Comment:
Arcade shows that energy free shinesparks + moonfall lets you farm this room pretty fast.